assets/styles/styles.css

```
@@ -0,0 +1,663 @@
+}
+
+p{
+  font-size: 1.125em;
+  line-height: 144%;
+  color: #26666B;
+}
+
+.center-content{
+  text-align: center;
+}
+
+.center-block{
+  margin: 0 auto;
+  display: block;
+}
+
+/*--------------------------------seccion NAV---------------------------------*/
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 26px 28px 0 28px;
+}
+
+.closed.nav_menu-toggle, 
+.closed .nav_logo,
+.closed .nav_phone-image {
+  filter: grayscale(0) brightness(0);
+}
+
+.nav_logo{
+  position: relative;
+  z-index: 2;
+}
+
+.nav_logo .nav_logo--big {
+  display: none;
+}
+
+/*------phone tooltip-----*/
+.nav_phone {
+  position: relative;
+  display: inline-block;
+}
+
+.nav_phone .nav_tooltip {
+  visibility: hidden;
+  width: 120px;
+  background-color: #000;
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 2;
+  top: -5px;
+  right: 105%;
+}
+
+.nav_phone:hover .nav_tooltip {
+  visibility: visible;
+}
+.nav_phone .nav_tooltip::after {
+  content: " ";
+  position: absolute;
+  top: 50%;
+  left: 100%; /* To the right of the tooltip */
+  margin-top: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent transparent transparent black;
+}
+/*-----end phone tooltip-----*/
+
+.nav_menu-toggle {
+  display: block;
+  position: relative;
+  -webkit-user-select: none;
+  user-select: none;
+  background: none;
+  border: 0 transparent;
+  outline: none;
+}
+
+.nav_menu-toggle a {
+  text-decoration: none;
+  color: #232323;
+  transition: color 0.3s ease;
+}
+
+.nav_menu-toggle span {
+  display: block;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;
+  background: #00343D;
+  border-radius: 3px;
+  z-index: 2;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+  background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+  opacity 0.55s ease;
+}
+
+.nav_menu-toggle span:first-child {
+  transform-origin: 0% 0%;
+}
+
+.nav_menu-toggle span:nth-last-child(2) {
+  transform-origin: 0% 100%;
+}
+
+.nav_menu-toggle.open span {
+  opacity: 1;
+  transform: rotate(45deg) translate(-10px, -10px);
+  background: #00D8A0;
+}
+
+.nav_menu-toggle.open span:nth-last-child(3) {
+  opacity: 0;
+  transform: rotate(0deg) scale(0.2, 0.2);
+}
+
+.nav_menu-toggle.open span:nth-last-child(2) {
+  transform: rotate(-45deg) translate(-10px, 10px);
+}
+
+.nav_menu {
+  position: absolute;
+  width: 100%;
+  margin: -26px 0 0 -28px;
+  padding: 100px 0 0 0;
+  height: calc(100% - 100px);
+  background: #fff;
+  list-style-type: none;
+  -webkit-font-smoothing: antialiased;
+  transform-origin: 0% 0%;
+  transform: translate(-100%, 0);
+  font-family: 'GTWalsheim','';
+  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0);
+  list-style-type: none;
+  z-index: 1;
+}
+
+.nav_menu li {
+  position: relative;
+  padding: 10px 0;
+  font-size: 1.2em;
+  text-align: center;
+  font-weight: bold;
+}
+
```

I see that you have some big blank spaces like this one. These spaces don't help with your code readability

---

i mostly leave big spaces to separate sections so they are visible on the scroll's zoom

i will remove them once im finished. or is there a better way of visualy separating css rules in the same file?

---

You can use comments for that. For instance
```suggestion
/**
Nav section
**/
```
And you can have the readability you want